tests/test_trainer_callback.py

```diff
@@ -221,3 +221,10 @@ def test_event_flow(self):
         trainer.train()
         events = trainer.callback_handler.callbacks[-2].events
         self.assertEqual(events, self.get_expected_events(trainer))
+
+        # warning should be emitted for duplicated callbacks
+        with unittest.mock.patch("transformers.trainer_callback.logger.warn") as warn_mock:
+            trainer = self.get_trainer(
+                callbacks=[MyTestTrainerCallback, MyTestTrainerCallback],
+            )
+            assert str(MyTestTrainerCallback) in warn_mock.call_args[0][0]
```

Comment:
The warning message emitted here looks like:

```
You are adding a <class 'tests.test_trainer_callback.MyTestTrainerCallback'> to
the callbacks of this Trainer, but there is already one. The currentlist of callbacks is
:DefaultFlowCallback
TensorBoardCallback
MyTestTrainerCallback
```

Reply:
`call_args[0][0]` corresponds to the first positional argument.

Reply:
Looks good!